### DISCUSSION TITLE
RSS feed prepending page URL to image URLs in the description. Feedly shows no images!

### DISCUSSION THREAD

**Discussion by rohitfarmer** [quarto-dev/quarto-cli#6943] [Q&A] [answered]
### Description

Hi, my RSS feeds do not display any images. All the images in my blog are embedded via URLs from Imgur or Unsplash. I just noticed that the image source URLs are prepended with the blog post URL in the feed description. Is it a bug or am I doing something wrong?

Example
```
<img src="https://outdoorsyindians.com/posts/002-00-a-beginners-guide-to-kayaking/https:/i.imgur.com/2YACDdt.jpeg" class="preview-image img-fluid figure-img">
```

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by rohitfarmer** [reply]
Hi, this is the repository I am working with. https://github.com/rohitfarmer/outdoorsy-indians-quarto

It hs the code and the site that was built on 1.3.X version. In my local copy, I am running the pre-release. But I am seeing this in both version. In my Feedly, I don't see any images that I have embedded, but it shows the embedded Google Maps.

**Reply by rohitfarmer** [reply]
The website is served here https://outdoorsyindians.com/ with three independent RSS feeds

**Reply by rohitfarmer** [reply]
Here is the document that shows this behavior, amongst others. I also noticed that I am not getting figure alt text by rendering with the pre-release version. 

````qmd

---
title: "Essentials for Outdoor Survival"
pagetitle: "{{< meta title >}}"
description: "Yet another guide to tools and skills required for outdoor survival"
keywords: Survival Gear List, Outdoor Survival Skills, The 5 Cs of Survival
categories: [Survival, First Aid, Cutting, Combustion, Cover, Container, Cordage, Communication, Navigation, Insect Repellent]
author: 
  - name: "Rohit Farmer"
date: "2023-08-24"
image: https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg
format: html
---

![Photo by Caspian Dahlström on Unsplash](https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg){fig-alt="Photo of a bushcraft shelter" .preview-image}

Text for the blog post ..... blah blah
````

**Reply by dragonstyle** [reply, marked as answer]
This will be resolved in our next pre-release build. Thanks for letting us know about the issue!

https://github.com/quarto-dev/quarto-cli/commit/db3037190ae0c759268940d7106b91e1b80ef500
